### PR TITLE
Inspec in docker

### PR DESCRIPTION
**Pull request by VolkerBohne**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+build-docker-images: ## custom Docker image with additional batteries included
+	docker build \
+		--build-arg INSPEC_VERSION=$(INSPEC_VERSION) \
+		-f docker/Dockerfile-inspec \
+		-t inspec_git_crypt:$(INSPEC_VERSION) .
+
+test: ## runs InSpec tests
+	@echo "Please be patient - this can take a while ......."
+	@-export $(shell aws-vault exec -n $(AWS_PROFILE) -- env | grep --color=never ^AWS | xargs) && \
```
Comment (michaelkrieg) [inline Makefile]: is this intended here? aws-vault could be run outside the makefile

**Comment by michaelkrieg** [comment]
could you please update the README accordingly?

File: Makefile (hunk 1 above)
Comment (VolkerBohne) [reply Makefile]: done

**Review by michaelkrieg — APPROVED** [review]
lgtm

**Pull request merged**